README.md

```diff
@@ -109,7 +109,7 @@ Now you can run tests as shown below:
 tox
 ```
 
-or, you can run them for a specific environment `tox -e python3.12-django5.0-wagtail6.0` or specific test
-`tox -e python3.12-django5.0-wagtail5.2 -- tests.test_file.TestClass.test_method`
+or, you can run them for a specific environment `tox -e python3.12-django5.0-wagtail6.1` or specific test
+`tox -e python3.12-django5.0-wagtail6.1 -- tests.test_file.TestClass.test_method`
```

Comment:
please leave the LTS here

Reply:
Retained: https://github.com/zerolab/wagtail-periodic-review/pull/21/commits/906460c70cf8589423602110a0f8a817d355dd22